### PR TITLE
Update system76 driver arch

### PR DESCRIPTION
**Pull request by ahoneybun**
This adds a new article for installing our software on Fedora and Arch, it also updates the System76 Driver article to install the firmware daemon before the System76 Driver.

### CODE REVIEW

**Comment by Seibz** [comment]
This looks good

File: _articles/system76-software.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  - support
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: hardware-drivers
+
+---
+
+## Arch
+
+Be sure to install the <u>System76 Driver</u> first and the steps to do that are [here](/articles/system76-driver).
```
Comment (storyaddict) [inline _articles/system76-software.md:21]: Make this two sentences:

Be sure to install the <u>System76 Driver</u> first. The steps to do that are [here]

**Review by storyaddict — CHANGES_REQUESTED** [review]
Fixed a sentence error.

File: _articles/system76-software.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ section: hardware-drivers
 
 ---
 
+# Notes about these instructions
```
Comment (storyaddict) [inline _articles/system76-software.md:15]: NOTE: These instructions were tested on a Galago Pro (galp3-b) and Pangolin (pang10). Neither system have NVIDIA GPUs, so this process doesn't go over installing the NVIDIA driver (system76-driver-nvidia).
Due to these limitations, switchable graphics may not work on NVIDIA systems.

**Review by storyaddict — CHANGES_REQUESTED** [review]

**Review by storyaddict — APPROVED** [review]
looks good

**Pull request merged**